### PR TITLE
single file component: AndOptions

### PR DESCRIPTION
**Pull request by bafolts**
This converts one of the components I didn't think would ever use a template. I added a unit test given it isn't as straight forward as other components.

### CODE REVIEW

**Comment by kberg** [comment]
The file move didn't occur. Looks like delete/add.

File: tests/components/AndOptions.spec.ts [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+        },
+        showsave: true,
+        showtitle: true,
+      },
+    });
+    const buttons = component.findAllComponents({name: 'Button'});
+    await buttons.at(0).findAllComponents({
+      name: 'button',
+    }).at(0).trigger('click');
+    expect(savedData).to.deep.eq([['1'], ['1']]);
```
Comment (kberg) [inline tests/components/AndOptions.spec.ts:43]: FYI, I think .eql is the same as .to.deep.eq. Don't bother, but I do think it's nice.

**Review by kberg — APPROVED** [review]
Much nicer!

File: src/components/AndOptions.vue [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+  methods: {
+    ...TranslateMixin.methods,
+    playerFactorySaved: function(idx: number) {
+      return (out: Array<Array<string>>) => {
+        console.log('setting the value to', out[0]);
+        this.$data.responded[idx] = out[0];
+      };
+    },
+    saveData: function() {
+      console.log(this);
```
Comment (bafolts) [inline src/components/AndOptions.vue:68]: Will remove before flight, was debugging.

**Pull request merged**